functions/New-DbaAgentJob.ps1

```diff
@@ -0,0 +1,476 @@
+            }
+        })]
+        [string]$PageOperatorName,
+        [Parameter(Mandatory = $false)]
+        [ValidateSet(0, "Never", 1, "OnSuccess", 2, "OnFailure", 3, "Always")]
+        [object]$DeleteLevel = '',
+        [bool]$Force,
+        [switch]$Silent
+    )
+
+    BEGIN
+    {
+        # Check the input for the job id or job name
+        if($JobName.Length -lt 1)
+        {
+            Stop-Function -Message "Please enter a job name." -Silent $Silent -InnerErrorRecord $_ -Target $JobName
+            return
+        }
+
+        # Check of the event log level is of type string and set the integer value
+        if(($EmailLevel.GetType().Name -eq 'String') -and ($EventLogLevel.Length -ge 1))
+        {
+            switch ($EventLogLevel)
+            {
+                {($_ -eq "Never") -or ($_ -eq '')} { $EventLogLevel = 0 } 
+                "OnSuccess" { $EventLogLevel = 1 } 
+                "OnFailure" { $EventLogLevel = 2 }
+                "Always" { $EventLogLevel = 3 } 
+            }
+        }
+
+        # Check of the email level is of type string and set the integer value
+        if(($EmailLevel.GetType().Name -eq 'String') -and ($EmailLevel.Length -ge 1))
+        {
+            switch ($EmailLevel)
+            {
+                {($_ -eq "Never") -or ($_ -eq '')} { $EmailLevel = 0 } 
+                "OnSuccess" { $EmailLevel = 1 } 
+                "OnFailure" { $EmailLevel = 2 }
+                "Always" { $EmailLevel = 3 } 
+            }
+        }
+
+        # Check of the net send level is of type string and set the integer value
+        if(($NetsendLevel.GetType().Name -eq 'String') -and ($NetsendLevel.Length -ge 1))
+        {
+            switch ($NetsendLevel)
+            {
+                {($_ -eq "Never") -or ($_ -eq '')} { $NetsendLevel = 0 } 
+                "OnSuccess" { $NetsendLevel = 1 } 
+                "OnFailure" { $NetsendLevel = 2 }
+                "Always" { $NetsendLevel = 3 } 
+            }
+        }
+
+        # Check of the page level is of type string and set the integer value
+        if(($PageLevel.GetType().Name -eq 'String') -and ($PageLevel.Length -ge 1))
+        {
+            switch ($PageLevel)
+            {
+                {($_ -eq "Never") -or ($_ -eq '')} { $PageLevel = 0 } 
+                "OnSuccess" { $PageLevel = 1 } 
+                "OnFailure" { $PageLevel = 2 }
+                "Always" { $PageLevel = 3 } 
+            }
+        }
+
+        # Check of the delete level is of type string and set the integer value
+        if(($DeleteLevel.GetType().Name -eq 'String') -and ($DeleteLevel.Length -ge 1))
+        {
+            switch ($DeleteLevel)
+            {
+                "Never" { $DeleteLevel = 0 } 
+                "OnSuccess" { $DeleteLevel = 1 } 
+                "OnFailure" { $DeleteLevel = 2 }
+                "Always" { $DeleteLevel = 3 } 
+            }
+        }
+
+        [object[]]$JobsCreated = $null
+    }
+
+    PROCESS
+    {
```

Comment:
I agree with @FriedrichWeinmann 

I now see the only example submitting multiple instances is indeed through the pipeline.
That leads to this next request 😄 : @sanderstad , can you add an example supplying multiple instances at the -SqlInstance parameter, please?
and also mention in the comment based help that SqlInstance accepts more than one?